sql_files/queries.sql

```diff
@@ -9,3 +9,71 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
 
+BEGIN;
+UPDATE animals SET species = 'digimon';
+WHERE name LIKE '%mon';
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'pokemon'
+WHERE species IS NULL OR species = '';
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
```

Comment:
- [ ] This is a close one! but please make sure to update the species to "digimon" for animals whose names end in "mon" alongside the query in line `19-20`.

Reply:
@AbelG101 I think i just forgot to include the transaction where i set the species to unspecified, and then rollback, as the following requirements are to set the digimons and one after is the pokemons, am i understanding this correctly? 

as i have 2 transactions shown, where i supposed to have 3?

because from the comments i fell like i should edit all 3 transactions, but i believe i just missed one.

Reply:
Hey @agneta1991 sorry for the late reply but what I meant was there have to be 2 updates, so just like you did for the Pokemon you'd need to include another update for setting the species to Digimon for all animals that have a name ending in mon.

  ![image](https://github.com/agneta1991/vet_clinic/assets/104968236/5f8e7d35-5ab4-42bb-8cb4-cd2a92d6f1b3)